File: server/runningServer/src/main/java/com/server/running/running/service/RunningService.java
package com.server.running.running.service;

import java.util.List;

import com.server.running.running.dto.Running;
import com.server.running.user.dto.User;

public interface RunningService {
	// 러닝 시작
	public Running start(Running running);
	
	// 러닝 종료
	public Running end(Running running);

}

File: client/app/src/main/java/com/example/runningapplication/service/RunningService.kt
package com.example.runningapplication.service

import com.example.runningapplication.data.model.Running
import com.example.runningapplication.data.model.User

import retrofit2.http.*
import retrofit2.Call

interface RunningService {

    @POST("/start.run")
    fun startRunning(
        @Body parameters: HashMap<String, Int>
    ):Call<Running>

    @PUT("/end.run")
    fun endRunning(
        @Body parameters: HashMap<String, Any>
    ):Call<Running>

    @GET("/findRunning.run")
    fun findRunning(
        @Query("uid") uid: Int
    ):Call<User>

    @GET("/distance.run")
    fun distance(
        @Query("uid") uid: Int
    ):Call<Double>
}